## Use /bin/sh to execute external programs instead of getting
## the shell from $SHELL (See ticket #388)
#
# http://trac.lighttpd.net/trac/ticket/388

--- lighttpd-1.4.12/src/proc_open.c~	2006-09-19 15:49:41.846531074 +0300
+++ lighttpd-1.4.12/src/proc_open.c	2006-09-19 15:51:13.658593480 +0300
@@ -219,11 +219,7 @@
 /* {{{ proc_open */
 int proc_open(proc_handler_t *proc, const char *command) {
 	pid_t child;
-	const char *shell;
-
-	if (NULL == (shell = getenv(SHELLENV))) {
-		shell = "/bin/sh";
-	}
+	const char *shell = "/bin/sh";
 
 	if (proc_open_pipes(proc) != 0) {
 		return -1;
